core/lib/dependency-graph/simulator/network-analyzer.js

```diff
@@ -126,9 +126,9 @@ class NetworkAnalyzer {
 
       // If the request was SSL we get two estimates, one for the SSL negotiation and another for the
       // regular handshake. SSL can also be more than 1 RT but assume False Start was used.
-      if (timing.sslStart > 0 && timing.sslEnd > 0) {
+      if (timing.sslStart >= 0 && timing.sslEnd >= 0) {
         return [timing.connectEnd - timing.sslStart, timing.sslStart - timing.connectStart];
-      } else if (timing.connectStart > 0 && timing.connectEnd > 0) {
+      } else if (timing.connectStart >= 0 && timing.connectEnd >= 0) {
         return timing.connectEnd - timing.connectStart;
```

Comment:
This is more correct (-1 is what signifies a missing value here, though in practice you would never see these timings be 0 unless you learned how to break the speed of light and sold amazing networking gear to everyone between you and the server being tested).

I changed to be `>=` here so that the faked values from LR can just use 0 as a baseline.

Reply:
Actually, this may be a real bug fix!

From a test failure using a real trace...
<img width="1212" alt="image" src="https://github.com/GoogleChrome/lighthouse/assets/4071474/873cf36c-bd54-49fd-a651-5ca7add7cedc">

looks like it is possibly for dns resolution to take no time at all ... and for a tcp connection to already be present ... but SSL not already established. So connect and ssl start times would be zero in that case. I think we may then be assuming that the TCP setup sample is 0ms, which is just....not possible, right? Maybe we drop 0s in this function?

Reply:
Clearly a fix needs to be made here for requests with 0 timing, but is it possible to separate it into another change? These are two pretty tricky issues and will be easier to evaluate isolated from each other.